LabBuilder/LabBuilder.psm1

```diff
@@ -1438,6 +1438,120 @@ function Get-LabVMTemplates {
     Return $VMTemplates
 } # Get-LabVMTemplates
 ####################################################################################################
+####################################################################################################
+<#
+.SYNOPSIS
+   Gets an Array of TemplateDisks for a Lab configuration.
+.DESCRIPTION
+   Takes a provided Lab Configuration file and returns the list of Template Disks
+   that will be used to create the Virtual Machines in this lab. This list is usually passed to
+   Initialize-LabVMTemplateDisks.
+.PARAMETER Configuration
+   Contains the Lab Builder configuration object that was loaded by the Get-LabConfiguration object.
+.EXAMPLE
+   $VMTemplateDisks = Get-LabVMTemplateDisks -Config c:\mylab\config.xml 
+   Loads a Lab Builder configuration and pulls the array of TemplateDisks from it.
+.OUTPUTS
+   Returns an array of TemplateDisks.
+#>
+function Get-LabVMTemplateDisks {
+    [OutputType([System.Collections.Hashtable[]])]
+    [CmdLetBinding()]
+    param
+    (
+        [Parameter(
+            Mandatory,
+            Position=0)]
+        [ValidateNotNullOrEmpty()]
+        [XML] $Config
+    )
+
+    [System.Collections.Hashtable[]] $VMTemplateDisks = @()
+    [String] $VHDParentPath = $Config.labbuilderconfig.SelectNodes('settings').vhdparentpath
+   
+	Write-Verbose "VHD parent path = $VHDParentPath" 
```

Comment:
Couple of things here (also applies to line 1498):
1. Rather than using positional parameters, can we use named parameters (PS Script Analyzer throws warnings up for positional parameters). E.g.
```powershell
Write-Verbose -Verbose "VHD parent path..."
```
2. Can the line indentation be corrected (many editors will correct this when they open the file and then Git will think it has been changed).
3. Can the message be more explicit about what is being reported? E.g. "Getting Template Disks in VHD Parent Path ..." and "VHD Template Disk $DiskName found."
4. Can we use the LocalizationData for the messages? I'm planning to move the LocalizationData out to a separate file soon (see #32). E.g.
 ```powershell
              Write-Verbose -Message $($LocalizedData.GettingTemplateDisksinVHDParentPathMessage`
                 -f $VHDParentPath)
```

Sorry to be so particular here. I have been doing a lot of work lately to try and get the code up to MS best practice guidelines.